packages/core/integration-tests/test/html.js

```diff
@@ -304,6 +304,10 @@ describe('html', function () {
         name: 'logo.svg',
         assets: ['logo.svg'],
       },
+      {
+        name: 'logo.14afa1de.svg',
```

Comment:
I'm sure theres a better way to do this, but I'm not sure what's good in the test suite for that.

Reply:
```suggestion
        type: 'svg',
```